deploy/roles/alertmanager-clusterrole_binding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: authorization.openshift.io/v1
+groupNames: null
+kind: ClusterRoleBinding
+metadata:
+  name: alertmanager-application-monitoring
+roleRef:
+  name: alertmanager-application-monitoring
+subjects:
+- kind: ServiceAccount
+  name: alertmanager
+  namespace: middleware-monitoring
```

Comment:
Are these files just used for testing purposes?

Reply:
Yes, they are in the repo for reference or when you want to deploy the operator outside of integreatly. But the operator itself doesn't use them.